.github/workflows/sync-labels.yaml

```diff
@@ -0,0 +1,43 @@
+name: label-syncer
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 12 * * *'
+
+permissions: read-all
+
+jobs:
+  label-syncer:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846 # v2.4.0
+
+      - uses: octo-sts/action@6177b4481c00308b3839969c3eca88c96a91775f # v1.0.0
+        id: octo-sts
+        with:
+          scope: chainguard-dev
+          identity: sync-labels
+
+      - name: run sync-labels
+        uses: docker://ghcr.io/cpanato/github-label-syncer:0.7.0@sha256:48969b80bcf741c5fee1553f584b8930a0b09a45b87f578bd06bf4f6f7a4f273
```

Comment:
What is this?  Is this something we should more into a repo in the org and/or regularly publish?

Could this be a Wolfi package and leverage `wolfi-act` for an ephemeral image?

Just wondering how we maintain this sustainably.

Reply:
spoke via slack, will wait your feedback to move forward